src/support/has.ts

```diff
@@ -62,10 +63,19 @@ add('es6-set', () => {
 /* Map */
 add('es6-map', function () {
 	if (typeof global.Map === 'function') {
-		/* IE11 and older versions of Safari are missing critical ES6 Map functionality */
-		const map = new global.Map([ [0, 1] ]);
-		return map.has(0) && typeof map.keys === 'function' &&
-			typeof map.values === 'function' && typeof map.entries === 'function';
+		/*
+		IE11 and older versions of Safari are missing critical ES6 Map functionality
+		We wrap this in a try/catch because sometimes the Map constructor exists, but does not
+		take arguments (iOS 8.4)
+		 */
+		try {
+			const map = new global.Map([ [0, 1] ]);
+			return map.has(0) && typeof map.keys === 'function' &&
+				typeof map.values === 'function' && typeof map.entries === 'function';
+		}
+		catch (e) {
+			return false;
```

Comment:
Let's add Istanbul ignore here, because now we aren't running CI where this will cause issues and then it will maintain the coverage.